django_feature_flip/tests.py

```diff
@@ -78,8 +106,68 @@ def test_activate_group_for_non_existent_feature(self):
 
     def test_activate_group_for_non_existent_group(self):
         FeatureFactory(name='my_feature')
-
         try:
             self.feature_flip.activate_group('my_feature', 'non_existent_group')
         except FeatureFlip.FeatureFlipError as error:
             self.assertEqual(error.message, 'Group non_existent_group does not exist')
+
+    def test_enabled_for_enabled_actor(self):
+        feature = FeatureFactory()
+        actor = ActorFactory(flip_id=1)
+        feature.actors.add(actor)
+        actor_object = self._actor(1)
+
+        self.assertTrue(self.feature_flip.enabled(feature.name, actor_object))
+
+    def test_enabled_for_disabled_actor(self):
+        feature = FeatureFactory()
+        actor = ActorFactory(flip_id=1)
+        feature.actors.add(actor)
+        actor_object = self._actor(2234)
```

Comment:
Is 2234 a number for any particular reason? Because if it's random you should generate it randomly.

Reply:
Should not be random, just should be an actor that does not exist